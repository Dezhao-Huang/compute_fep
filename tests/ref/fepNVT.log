LAMMPS (12 Apr 2014)
  orthogonal box = (-9.36391 -9.36391 -9.36391) to (9.36391 9.36391 9.36391)
  1 by 1 by 2 MPI processor grid
  reading atoms ...
  649 atoms
  reading velocities ...
  649 velocities
  scanning bonds ...
  1 = max bonds/atom
  scanning angles ...
  1 = max angles/atom
  reading bonds ...
  432 bonds
  reading angles ...
  216 angles
  2 = max # of 1-2 neighbors
  1 = max # of 1-3 neighbors
  1 = max # of 1-4 neighbors
  2 = max # of special neighbors
  0 = # of size 2 clusters
  0 = # of size 3 clusters
  0 = # of size 4 clusters
  216 = # of frozen angles
PPPM initialization ...
  G vector (1/distance) = 0.266541
  grid = 10 10 10
  stencil order = 5
  estimated absolute RMS force accuracy = 0.0275322
  estimated relative force accuracy = 8.29123e-05
  using double precision FFTs
  3d grid and FFT values/proc = 3468 500
FEP settings ...
  temperature = 300.000000
  tail no
  lj/cut/coul/long sigma 1-1 3-3
  1-1 charge
Memory usage per processor = 4.68629 Mbytes
Step TotEng PotEng E_vdwl E_coul E_long Temp lambda sig13 q1 
       0   -2000.7075   -2375.9175    444.27231    8806.8609   -11627.051    291.37777            0         2.78            0 
      30   -1999.3264   -2359.5129    460.48483    8804.1775   -11624.175    279.71093          0.1        2.835          0.1 
      60   -1994.2453   -2374.8586    462.96264    8790.0298   -11627.851    295.57379          0.2         2.89          0.2 
      90   -1990.8794   -2393.0178    466.82167    8766.7511   -11626.591    312.28956          0.3        2.945          0.3 
     120    -1986.446   -2375.7327     482.6205     8769.722   -11628.075    302.30933          0.4            3          0.4 
     150   -1984.1238   -2397.2327    456.33747    8773.8783   -11627.448    320.80896          0.5        3.055          0.5 
     180   -1981.9248   -2374.6906    441.89157    8809.9476    -11626.53     305.0111          0.6         3.11          0.6 
     210   -1978.1756   -2376.9561    403.46725    8846.7401   -11627.163    309.68194          0.7        3.165          0.7 
     240   -1973.4958   -2389.7243    415.40455    8820.6141   -11625.743    323.23154          0.8         3.22          0.8 
     270   -1969.5197   -2400.6225    440.70736    8784.5437   -11625.874    334.78255          0.9        3.275          0.9 
     300   -1970.8491   -2387.4958    479.23301    8753.9027   -11620.632    323.55631            1         3.33            1 
Loop time of 2.75521 on 2 procs for 300 steps with 649 atoms
