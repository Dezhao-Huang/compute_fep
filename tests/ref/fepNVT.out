# Time-averaged data for fix fFEP
# TimeStep c_cFEP[1] c_cFEP[2]
30 1.99105 0.0404927
60 0.409437 0.524396
90 0.333532 0.6471
120 -0.623469 5.47087
150 -0.508445 2.62078
180 -1.46687 12.1654
210 -2.58389 77.9138
240 -1.49649 12.6949
270 -1.4576 15.9517
300 -1.32374 11.0942
